pkg/api/v1beta1/types.go

```diff
@@ -411,9 +411,11 @@ type Container struct {
 	Name string `json:"name" description:"name of the container; must be a DNS_LABEL and unique within the pod; cannot be updated"`
 	// Required.
 	Image string `json:"image" description:"Docker image name"`
-	// Optional: Defaults to whatever is defined in the image.
-	Command []string `json:"command,omitempty" description:"command argv array; not executed within a shell; defaults to entrypoint or command in the image; cannot be updated"`
-	// Optional: Defaults to Docker's default.
+	// Optional: The image's entrypoint is used if this is not provided.
+	Entrypoint []string `json:"entrypoint:omitempty" description:"entrypoint array; not executed within a shell; the image's entrypoint is used if this is not provided"`
```

Comment:
@bgrant0607 I believe this is actually not the case; `PrepareForUpdate` only prevents the status from being updated:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/registry/pod/rest.go#L63

I will make it so that you cannot update these fields.